dataprofiler/tests/profilers/test_base_option.py

```diff
@@ -17,12 +17,21 @@ class TestBaseOption(unittest.TestCase):
 	def setUpClass(cls):
 		cls.data = Data(data=pd.DataFrame([1, 2]), data_type='csv')
 	
+
+	@classmethod
+	def getOptions(self, **params):
+		return BaseOption(**params)
```

Comment:
I think it might be better if we use a class variable o define what get options is such that the new tests requires it to be defined. Otherwise, anything that imports this class could forget to alter the getOptions and end up with the wrong unittests. Additionally, it might be best to create it as an abstract test set which then forces the setting.

i.e.
```python

class AbstractTestOptions...(object):
    
    option_class = None
    
    @classmethod
    def get_options(self, *args, **kwargs):
        if not self.option_class:
            raise ....
        return self.option_class(*args, **kwargs)

    .
    .
    . <other abstract tests which are universal>


class TestBaseOptions(AbstractTestOptions, unittest.....):

    option_class = BaseOption
    
    .
    .
    . <Tests etc...>
```

Reply:
Sure, I can create an abstract class that is implemented by the other TestOptions classes and have it check and enforce that setting.